projects/plugins/jetpack/extensions/blocks/sharing-buttons/block.json

```diff
@@ -0,0 +1,32 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 1,
+	"name": "jetpack/sharing-buttons",
+	"title": "Sharing Buttons",
+	"description": "Sharing Buttons.",
+	"keywords": [ "sharing", "social" ],
+	"version": "12.5.0",
+	"textdomain": "jetpack",
+	"category": "embed",
+	"icon": "<svg viewBox='0 0 24 24' xmlns='http://www.w3.org/2000/svg'><path d='M9 11.8l6.1-4.5c.1.4.4.7.9.7h2c.6 0 1-.4 1-1V5c0-.6-.4-1-1-1h-2c-.6 0-1 .4-1 1v.4l-6.4 4.8c-.2-.1-.4-.2-.6-.2H6c-.6 0-1 .4-1 1v2c0 .6.4 1 1 1h2c.2 0 .4-.1.6-.2l6.4 4.8v.4c0 .6.4 1 1 1h2c.6 0 1-.4 1-1v-2c0-.6-.4-1-1-1h-2c-.5 0-.8.3-.9.7L9 12.2v-.4z' /></svg>",
+	"supports": {
+		"align": false,
+		"alignWide": true,
+		"anchor": false,
+		"customClassName": true,
+		"className": true,
+		"html": false,
+		"multiple": false,
```

Comment:
Any tehnical blockers letting the block be in the post or template multiple times?

Seems handy if they want to have the sharing e.g. on top of the post, and in the footer.

Even in the middle section for some reason, why not! ;-)